_i18n/en/resources/user-guides/monero-wallet-cli.md

```diff
@@ -1,79 +1,107 @@
 
-### Specify the number of outputs for a transaction:
+## Sending monero
 
-    transfer RINGSIZE ADDRESS AMOUNT
+You will need the standard address you want to send to (a long string starting with '4' or a '8'). The  command structure is:
 
-Replace `RINGSIZE` with the number of outputs you wish to use. **If not specified, the default is 11.** It's a good idea to use the default, but you can increase the number if you want to include more outputs. The higher the number, the larger the transaction, and higher fees are needed.
+```
+transfer ADDRESS AMOUNT
+```
 
+Replace `ADDRESS` with the address you want to send to and `AMOUNT` with how many monero you want to send.
 
 ## Receiving monero
 
-If you have your own Monero address, you just need to give your standard address to someone.
+If you have your own Monero address, you just need to give your address to someone.
 
-You can find out your address with:
+You can find out your primary address with:
 
-    address
+```
+address
+```
 
 Since Monero is anonymous, you won't see the origin address the funds you receive came from. If you
 want to know, for instance to credit a particular customer, you'll have to tell the sender to use
-a payment ID, which is an arbitrary optional tag which gets attached to a transaction. To make life
-easier, you can generate an address that already includes a random payment ID:
+a payment ID, which is an arbitrary optional tag which gets attached to a transaction. It's not possible to use standalone payment addresses, but you can generate an address that already includes a random payment ID (integrated addresss) using `integrated_address`:
 
-    integrated_address
+```
+Random payment ID: <82d79055f3b27f56>
+Matching integrated address: 4KHQkZ4MmVePC2yau6Mb8vhuGGy8LVdsZD8CFcQJvr4BSTfC5AQX3aXCn5RiDPjvsEHiJu1TC1ybR8pRTCbZM5bhTrAD3HDwWMtAn1K7nV
+```
 
 This will generate a random payment ID, and give you the address that includes your own account
-and that payment ID. If you want to select a particular payment ID, you can do that too:
+and that payment ID. If you want to select a particular payment ID, you can do that too. Use:
 
-    integrated_address 12346780abcdef00
+```
+integrated_address 82d79055f3b27f56
+```
 
 Payments made to an integrated address generated from your account will go to your account,
 with that payment id attached, so you can tell payments apart.
 
+### Using subaddresses
+
+It's suggested to use subaddresses (starting with `8`) instead of your main address (starting with `4`) to receive funds. To generate a new subaddress, run:
+
+```
+address new
+```
+
+This will generate a subaddress, whcih you can share to receive payment on the account it's linked to. To view all generated addresses, run:
```

Comment:
```suggestion
This will generate a subaddress, which you can share to receive payment on the account it's linked to. To view all generated addresses, run:
```
Side note: right before the command, it says `To generate a new subaddress, run:`, and then right after is `This will generate a subaddress`. It feels a little redundant. Maybe remove the beginning portion?

Reply:
Good point. Removed `To generate a new subaddress,`